### PR TITLE
test

### PR DESCRIPTION
**Pull request by OleksandrFEdev**
- [DEMO LINK](https://OleksandrFEdev.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://OleksandrFEdev.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
+    rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img class="card__img"
+      src="images/imac.jpeg"
+      alt="PC"
+      >
+      <h2 class="card__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <div class="card__product">Product code: 195434</div>
```
Comment (nazarmatsevych) [inline src/index.html:21]: Use semantic tags for text content like `<p>` or `<h3>` and so on

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,42 @@
...
+
+      <div class="card__stars">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__stars-text">Reviews: 5</div>
```
Comment (nazarmatsevych) [inline src/index.html:31]: same here

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,42 @@
...
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-label">Price:</p>
+        <p class="card__money">$2,199</p>
+      </div>
+      <a class="card__button" href="#"
+      data-qa="hover">BUY</a>
```
Comment (nazarmatsevych) [inline src/index.html:39]:
```suggestion
      <a
        class="card__button"
        href="#"
        data-qa="hover"
      >
        BUY
      </a>
```

if you have 3 or more attributes then each attribute should be on a new line

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,115 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-secondary-color);
+  --main-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
```
Comment (nazarmatsevych) [inline src/style.css:8]: You can move variables to :root selector to make them global

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+.card {
+  box-sizing: border-box;
```
Comment (nazarmatsevych) [inline src/style.css:12]: move box-sizing to * selector

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+  line-height: 18px;
+  margin: 0;
+}
+
+.card__money {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: bold;
+  color: var(--text-accent-color);
+  margin: 0;
```
Comment (nazarmatsevych) [inline src/style.css:92]: You can reset margins for `<p>` tag instead of setting it to each class

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+.stars,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+}
```
Comment (nazarmatsevych) [inline src/style.css:57]:
```suggestion
.stars {
  display: flex;
}
```

I guess it should work as well

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Good work,
pls check my comments and proofread your code with [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

**Review by slavanosatskyi — APPROVED** [review]
👍

**Pull request closed without merging**